aws/lambda/log-classifier/ruleset.toml

```
@@ -256,6 +256,10 @@ pattern = 'IMPROVED:\s+graph_breaks='
 name = 'npm error'
 pattern = '^npm ERR! code .*'
 
+[[rule]]
+name = 'ghstack merge check error'
+pattern = '[Mergeability Error] .*'
```

```suggestion
pattern = 'is NOT mergeable into main'
```

Wouldn't this already work? It looks like the ones that require the start to match exactly use a `^`.

---

I'd also note that `[]` are special characters in regex and need to be escaped.

---

Will just remove the `[]`. I rather just name the error and match against that as generally the content of an error should fungible imo